lib/gssapi/mech/gss_cred.c

```diff
@@ -195,6 +331,19 @@ gss_import_cred(OM_uint32 * minor_status,
 	buffer.value = data.data;
 	buffer.length = data.length;
 
+	if (m->gm_flags & GM_USE_MG_CRED) {
+	    if (gss_oid_equal(&m->gm_mech_oid, GSS_SPNEGO_MECHANISM))
```

Comment:
Given the above, do we reach this code ever?

Reply:
Yes.